src/index.js

```diff
@@ -49,28 +37,55 @@ const display = () => {
     input.id = task.id;
     input.checked = task.completed;
     /* eslint-disable */
-    input.addEventListener('change', () => {
-      completedStatus(task, arr);
+    input.addEventListener('change', (e) => {
+      completedStatus(e.target, e.target.parentNode.parentNode.getAttribute('data-id'), label);
     });
     /* eslint-enable */
     list.appendChild(input);
 
     const label = document.createElement('label');
     label.innerHTML = `${task.description}`;
     label.className = 'task-name';
+    edit(label);
```

Comment:
- [ ] You will need to refactor this part so that our suggested change above can work properly.

```suggestion
    label.innerHTML = `${task.description}`;
    label.className = 'task-name';
    label.setAttribute('contenteditable', 'true');
    label.setAttribute('id', index);
    edit(label);
```